routes/auth.js

```diff
@@ -56,6 +132,51 @@ router.post('/login', (req, res) => {
     })
 });
 
+// create token adminshop
+router.post('/login/admin', (req, res) => {
+    const formData = {
+        email: req.body.email,
+        password: req.body.password,
+    };
+
+    connection.query('SELECT * FROM adminshop WHERE email = ?', [formData.email], (err, user) => {
+        if (err) {
+            res.status(500).send(err);
+        } else {
+            const isSame = bcrypt.compare(req.body.password, user[0].password)
+            if (!isSame) {
+                res.status(500).send('Wrong password')
+            } else {
+                jwt.sign({ user }, 'secretkey', (err, token) => {
+                    res.json({ token })
+                })
+            }
+        }
+    })
+});
+
+//create token super-admin
+router.post('/login/superadmin', (req, res) => {
+    const formData = {
+        email: req.body.email,
+        password: req.body.password,
+    };
+
+    connection.query('SELECT * FROM admin WHERE email = ?', [formData.email], (err, user) => {
+        if (err) {
+            res.status(500).send(err);
+        } else {
+            const isSame = bcrypt.compare(req.body.password, user[0].password)
+            if (!isSame) {
+                res.status(500).send('Wrong password')
+            } else {
+                jwt.sign({ user }, 'secretkey', (err, token) => {
```

Comment:
Même chose pour la variable d'environnement
```suggestion
                jwt.sign({ user }, process.env.SECRET_KEY', (err, token) => {
```